.storybook/style.css

```
@@ -1,3 +1,10 @@
+@import url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap');
```

Stylelint (17.14.0)

[error] Expected "url('https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap')" to be "'https://fonts.googleapis.com/css2?family=Space+Grotesk:wght@300..700&display=swap'" (import-notation)

(import-notation)